package.json

```diff
@@ -116,37 +115,38 @@
     "@testing-library/jest-dom": "5.14.1",
     "@testing-library/react": "12.0.0",
     "@types/bootstrap": "4.5.0",
-    "@types/chart.js": "2.9.32",
-    "@types/enzyme": "3.10.8",
+    "@types/chart.js": "2.9.34",
+    "@types/enzyme": "3.10.10",
     "@types/highlight.js": "10.1.0",
-    "@types/history": "4.7.8",
+    "@types/history": "4.7.9",
     "@types/jest": "26.0.24",
     "@types/jquery": "3.5.5",
     "@types/katex": "0.11.0",
-    "@types/lodash": "4.14.170",
+    "@types/lodash": "4.14.177",
     "@types/luxon": "2.0.2",
-    "@types/mark.js": "8.11.5",
+    "@types/mark.js": "8.11.6",
     "@types/marked": "0.7.4",
     "@types/moment-timezone": "0.5.30",
     "@types/node": "16.3.1",
     "@types/react": "16.9.49",
-    "@types/react-beautiful-dnd": "^13.0.0",
+    "@types/react-beautiful-dnd": "^13.1.2",
```

Comment:
Not related to this PR but we should use fixed versions instead of `^`

Reply:
Totally agree on this point. how should we go about doing this @hmhealey

Reply:
If you just do `npm install --save-dev --exact @types/react-beautiful-dnd@13.1.2`, it should set that correctly. I'm surprised it wasn't set already since we have the `.npmrc` set up to prefer exact versions to begin with

Reply:
Yes i have removed the `^` from dependency, also checked other but none have the `^` or `~`.